projects/plugins/boost/app/assets/src/js/utils/get-isa-error-suggestion.ts

```diff
@@ -0,0 +1,11 @@
+import { __ } from '@wordpress/i18n';
+
+export default function getIsaErrorSuggestion( code ) {
+	let suggestion = '';
+
+	if ( 429 === code ) {
+		suggestion = __( 'Minimum time between requests is 1 hour.', 'jetpack-boost' );
```

Comment:
We may change the threshold in boost-cloud from time to time. So, it would be nice if the message was coming from the API instead.